rust/oak/Cargo.toml

```diff
@@ -8,6 +8,10 @@ protobuf = "*"
 fmt = "*"
 log = "*"
 byteorder = "1"
+rand_core = "0.5"
+
+[features]
+default = ["rand_core/std"]
```

Comment:
Should this be on rand_core itself? Something like `rand_core = { version = "0.5", features = ["std"] }`

Reply:
Done.